sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/SQLOperation.java

```diff
@@ -443,7 +456,7 @@ private SerDe getSerDe() throws SQLException {
   private HiveConf getConfigForOperation() throws HiveSQLException {
     HiveConf sqlOperationConf = getParentSession().getHiveConf();
     if (!getConfOverlay().isEmpty() || shouldRunAsync()) {
-      // clone the parent session config for this query
+      // clone the partent session config for this query
```

Comment:
Change has the sp LOL. But this is pulled in so we probably cannot help it

Reply:
:( I know... either we fix it in hive 1.2.1 repo or here, but here, future changes will surface again :(